### ISSUE TITLE
please add an Expand layer

### ISSUE THREAD

**Issue by trotsky1997** [arogozhnikov/einops#72]
Like torch.tensor.expand

**Comment by arogozhnikov** [comment]
`einops.repeat` already covers expand functionality (except underlying implementation relies on repeat, but this can be changed)

**Comment by trotsky1997** [comment]
> `einops.repeat` already covers expand functionality (except underlying implementation relies on repeat, but this can be changed)

yes i know that,but in repeat method you should point which dimension to repeat.
in expand method,only expanded shape was needed,it was more convinent.
thanks
